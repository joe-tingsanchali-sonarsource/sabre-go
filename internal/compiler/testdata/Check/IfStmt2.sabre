package main

func main() {
	if z := 1; z {
		z = 0
	}
}