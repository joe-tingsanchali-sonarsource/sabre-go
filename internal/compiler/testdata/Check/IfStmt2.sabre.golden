>> 		if z := 1; z {
>> 		           ^   
Error[internal/compiler/testdata/Check/IfStmt2.sabre:4:13]: if condition should be boolean, but found 'int'
